generator/src/main/java/com/scottlogic/deg/generator/restrictions/MatchesStandardStringRestrictions.java

```diff
@@ -40,6 +37,8 @@ private StringGenerator getStringGenerator() {
                 return new SedolStringGenerator();
             case CUSIP:
                 return new CusipStringGenerator();
+            case RIC:
```

Comment:
It looks like it would be cleaner to have these generators implement a common interface with the instance providing and length methods.

Reply:
I'm not sure I understand your comment, because I thought that that was already the case?